packages/http-server/test/integration/http-server.integration.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/http-server
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+import {RestServer, RestComponent} from '@loopback/rest';
+import {Application, ApplicationConfig} from '@loopback/core';
+import * as assert from 'assert';
+
+describe('HttpServer', () => {
+  it('starts server', async () => {
+    const server = await givenAServer();
+    await server.start();
+    assert(server.listening, 'Server not started');
+    await server.stop();
+  });
+
+  it('stops server', async () => {
+    const server = await givenAServer();
+    await server.start();
+    await server.stop();
```

Comment:
Similarly here - I think you need to verify that the server is no longer listening on the given port, e.g. a request fails.